templates/qr.mustache

```diff
@@ -75,6 +75,7 @@ along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
 {{#fullview}}
 <p class="text-center"><i class="fa fa-location-arrow"></i> <a href="https://kurzlinks.mebis.bayern.de/home?link={{urlshort}}">{{#str}} urlshortlabel, block_qr {{/str}}</a></p>
 {{/fullview}}
+{{#qrcodecontent}}
```

Comment:
As this file is becoming more complex, it would be nice if you could indent the lines properly so it becomes more readable and easier to see which lines are active in which cases.